.github/workflows/deploy-enterprise.yml

```diff
@@ -50,6 +52,7 @@ jobs:
             required_surplus: 0
             security_environment: ASF
             ami_id: /aws/service/ecs/optimized-ami/amazon-linux-2/recommended/image_id
+            instance_types: c5d.xlarge
```

Comment:
I've updated the GUNW deployments (this and next two) to use the cost-optimized instance type based on our optimization study.

Reply:
Costs have remained stable for these machines the last 3 months:
![image](https://user-images.githubusercontent.com/7882693/156304742-b8de592f-2ad7-4e10-9aaa-eeae0ba3bea6.png)